easybuild/easyconfigs/l/Libint/Libint-2.5.0-gompi-2019a.eb

```diff
@@ -0,0 +1,25 @@
+name = 'Libint'
+version = '2.5.0'
+
+homepage = 'https://github.com/evaleev/libint'
+description = """Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body
+ matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory."""
+
+toolchain = {'name': 'gompi', 'version': '2019a'}
+toolchainopts = {'pic': True, 'cstd': 'c++11'}
+
+source_urls = ['https://github.com/evaleev/libint/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['e57bb4546a6702fdaa570ad6607712f31903ed4618f051150979a31a038ce960']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('GMP', '6.1.2'),
+    ('Boost', '1.70.0'),
+    ('Eigen', '3.3.7', '', True),
+    ('Python', '2.7.15'),
```

Comment:
Can it use Python3 when building?

Reply:
@Micket it's not Python3 compatible (but you could make it compatible without too much effort if you really wanted).

Reply:
@smoors I started testbuilding it myself, and indeed, it seems like (at least) one of the tests have an uses python2 stuff. execfile. Libint itself happily uses python3 for the rest though. I will report a bug to them, and maybe they will fix it for the next release.

I think we can just stick with python2 for now.